test/check-application

```diff
@@ -3434,6 +3445,13 @@ ContainerName=guitarq1
             b.wait_text("#containers-containers [data-label='Owner']", "user: guitar")
             b.wait_visible("#containers-containers .ct-badge-service")
 
+            # No systemd service link is shown for quadlets from other users as
+            # the services page doesn't support anything except the logged in/admin/root user.
+            self.toggleExpandedContainer("guitarq1")
+            sel = "#containers-containers tbody tr:contains('guitarq1')"
+            b.wait_in_text(f"{sel} + tr", "sleep infinity")
+            b.wait_not_present(f"{sel} button:contains('guitarq1.service')")
```

Comment:
It seems a little more robust to check for the absence of the "Systemd service", but matter of taste.